### PR TITLE
Strong name signing

### PR DESCRIPTION
**Pull request by AnastasiaKubova**
Things to consider before you submit the PR:

* [x] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] ~Did you add unit tests if this modifies the Windows code?~
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Update modules for using the strong naming signing.

## Related PRs or issues

[AB#84482](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/84482)

### CODE REVIEW

File: scripts/strong-named-sign-delay.cmd [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+EXIT /B
+
+:registerOrUnregisterFiles
+echo "Start to processing the file: %~1"
+if exist %~1 (
+    sn -%action% "%~1"
+) else (
+    echo "File %~1 doesn't exist".
+)
+goto:EOF
```
Comment (MatkovIvan) [inline scripts/strong-named-sign-delay.cmd:40]: It's already end of file. Why is it required?

File: scripts/strong-named-sign-delay.cmd [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+call "%VS140COMNTOOLS%VsDevCmd.bat"
+
+call :registerOrUnregisterFiles %pathToFile%Microsoft.AppCenter.dll
+call :registerOrUnregisterFiles %pathToFile%Microsoft.AppCenter.Analytics.dll
+call :registerOrUnregisterFiles %pathToFile%Microsoft.AppCenter.Crashes.dll
+call :registerOrUnregisterFiles %pathToFile%Microsoft.AppCenter.Distribute.dll
+EXIT /B
+
+:registerOrUnregisterFiles
+echo "Start to processing the file: %~1"
```
Comment (DmitriyKirakosyan) [inline scripts/strong-named-sign-delay.cmd:34]:
```suggestion
echo "Start processing the file: %~1"
```

File: scripts/strong-named-sign-delay.cmd [hunk 3]
```diff
@@ -0,0 +1,39 @@
+:: Copyright (c) Microsoft Corporation. All rights reserved.
+:: Licensed under the MIT License.
+
+:: Note: This script is used to skip a stong-naming check for Puppet apps.
+
+:: This script uses the VS140COMNTOOLS environment variable which contains path 
+:: to the Visual Studio Tools. By default, the path should be like this 
+:: "C:\Program Files (x86)\Microsoft Visual Studio [version]\Common7\Tools\" 
+:: and should contains VsDevCmd.bat file.
```
Comment (DmitriyKirakosyan) [inline scripts/strong-named-sign-delay.cmd:9]:
```suggestion
:: and should contain VsDevCmd.bat file.
```

File: scripts/strong-named-sign-delay.cmd [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+:: If this VS140COMNTOOLS environment variable doesn't contains this file 
+:: you should replace this path with the right Visual Studio Tools folder 
+:: or some other path where places the VsDevCmd.bat file.
```
Comment (DmitriyKirakosyan) [inline scripts/strong-named-sign-delay.cmd:13]:
```suggestion
:: or some other path containing the VsDevCmd.bat file.  
```

**Review by DmitriyKirakosyan — DISMISSED** [review]
Tested locally, works for me!
Approved! Please update changelog.

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,6 +6,8 @@
 
 * **[Fix]** Fix SDK not sending events when application is packed to self-contained single-file executable.
 
+* **[Feature]** In the App Center SDK was added support of strong-named sign assemblies for compatibility with applications that use strong-named sign. Read more about [strong-named assemblies](https://docs.microsoft.com/en-us/dotnet/standard/assembly/strong-named).
```
Comment (MatkovIvan) [inline CHANGELOG.md:9]: Please move feature above fix and remove ampty line between them

**Review by MatkovIvan — APPROVED** [review]

**Comment by EatonZ** [comment]
Very happy to see this finally.👍
Will 4.1.1 release tomorrow?

**Comment by MatkovIvan** [comment]
@EatonZ no, .NET will be released along other platforms later. No specific date.

**Comment by EatonZ** [comment]
@MatkovIvan Are you releasing soon?

**Comment by MatkovIvan** [comment]
@EatonZ a new version of the SDKs is planned to be released this month

**Pull request merged**